Mathlib/Algebra/Order/CauSeq/Completion.lean

```diff
@@ -267,16 +267,17 @@ theorem ofRat_inv (x : β) : ofRat x⁻¹ = ((ofRat x)⁻¹ : (Cauchy abv)) :=
     [simp only [const_limZero.1 h, GroupWithZero.inv_zero, const_zero]; rfl]
 #align cau_seq.completion.of_rat_inv CauSeq.Completion.ofRat_inv
 
-/- porting note: This takes a long time to compile.
```

Comment:
What is referring to? Is this why it is `noncomputable`?

Reply:
I have no idea what it refers to, but it seems pretty instant now. This is `noncomputable` because `inv` is noncomputable, as it uses a classical `if`; it's nothing to do with compilation performance.

Reply:
It looks like it was fixed long ago.